File: 1012.rs
use std::io;

fn main() {
  const PI: f64 = 3.14159;

  let mut input = String::new();

  io::stdin().read_line(&mut input).unwrap();

  let vec_input: Vec<&str> = input.split_whitespace().collect();

  let value_a = vec_input[0].parse::<f64>().unwrap();
  let value_b = vec_input[1].parse::<f64>().unwrap();
  let value_c = vec_input[2].parse::<f64>().unwrap();
   
  println!("TRIANGULO: {:.3}
CIRCULO: {:.3}
TRAPEZIO: {:.3}
QUADRADO: {:.3}
RETANGULO: {:.3}",
value_a * value_c / 2.0,
value_c * value_c * PI,
(value_a + value_b) * value_c / 2.0,
value_b *  value_b,
value_a * value_b)
}

File: Iniciante/1012/1012.java
import java.util.Scanner;

public class Main {
  public static void main(String[] args) {
    final double PI = 3.14159;

    Scanner scan = new Scanner(System.in);

    String[] values = scan.nextLine()
      .split(" ");

    double a = Double.parseDouble(values[0]);
    double b = Double.parseDouble(values[1]);
    double c = Double.parseDouble(values[2]);

    System.out.printf("TRIANGULO: %.3f\n", a * c / 2.0);
    System.out.printf("CIRCULO: %.3f\n", c * c * PI);
    System.out.printf("TRAPEZIO: %.3f\n", (a + b) * c / 2.0);
    System.out.printf("QUADRADO: %.3f\n", b *  b);
    System.out.printf("RETANGULO: %.3f\n", a * b);
  }
}